### ISSUE TITLE
Unable to hit web server

### ISSUE THREAD

**Issue by aanunez** [123andy/redcap-docker-compose#72]
Recently after using this repo for a while I tried spinning up a new container for a re-install (new laptop, new OS) and found that I was unable to access localhost (generic connection reset error). Nothing interesting is in the logs (below). When I tried a few other apache based docker containers everything seemed fine, so I assume its not a windows issue. I've left everything as the default in the .env and removed phpmyadmin from the compose file. Apache seems to be running in the web container and I can't find any obvious-to-me issue. Do you have any recommendations for troubleshooting this?

```
[Fri Apr 21 08:00:50.500925 2023] [mpm_prefork:notice] [pid 1] AH00170: caught SIGWINCH, shutting down gracefully
[Fri Apr 21 08:04:51.934394 2023] [mpm_prefork:notice] [pid 1] AH00163: Apache/2.4.38 (Debian) PHP/8.1.18 configured -- resuming normal operations
[Fri Apr 21 08:04:51.935140 2023] [core:notice] [pid 1] AH00094: Command line: '/usr/sbin/apache2 -D FOREGROUND'
```

**Comment by 123andy** [comment]
Sometimes I’ve had to use 127.0.0.1 or I typically use an entry in my hosts file so redcap.local resolves to 127.0.0.1.   Never figured out why this happens.
________________________________
From: Adam ***@***.***>
Sent: Friday, April 21, 2023 8:14:14 AM
To: 123andy/redcap-docker-compose ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [123andy/redcap-docker-compose] Unable to hit web server (Issue #72)


Recently after using this repo for a while I tried spinning up a new container for a re-install (new laptop, new OS) and found that I was unable to access localhost. Nothing interesting is in the logs (below). When I tried a few other apache based docker containers everything seemed fine, so I assume its not a windows issue. I've left everything as the default in the .env and removed phpmyadmin from the compose file. Apache seems to be running in the web container and I can't find any obvious-to-me issue. Do you have any recommendations for troubleshooting this?

[Fri Apr 21 08:00:50.500925 2023] [mpm_prefork:notice] [pid 1] AH00170: caught SIGWINCH, shutting down gracefully
[Fri Apr 21 08:04:51.934394 2023] [mpm_prefork:notice] [pid 1] AH00163: Apache/2.4.38 (Debian) PHP/8.1.18 configured -- resuming normal operations
[Fri Apr 21 08:04:51.935140 2023] [core:notice] [pid 1] AH00094: Command line: '/usr/sbin/apache2 -D FOREGROUND'

**Comment by aanunez** [comment]
I'm not sure what actually broke my docker deploy but after removing everything a few times and rebuilding everything its just suddenly working again. I assume it must be related to my windows install, but unfortunately I can't contribute anything for the next person that has this issue (if anyone). Feel free to close this.
